src/main/java/com/frcteam3255/components/swerve/SN_SuperSwerve.java

```diff
@@ -387,7 +398,7 @@ public Rotation2d getRotation() {
 			simAngle += swerveKinematics.toChassisSpeeds(lastDesiredStates).omegaRadiansPerSecond * timeFromLastUpdate;
 			return new Rotation2d(simAngle);
 		}
-		return Rotation2d.fromDegrees(pigeon.getYaw());
+		return Rotation2d.fromDegrees(pigeon.getYaw().getValue());
```

Comment:
is this a supplier that we want to reused?

Reply:
This method only really exists so that it's easier to make simulations. We could just get the pigeon yaw, but this isn't much harder and it forces students to use Rotation2ds (which have built-in unit conversion)